nssp_demo/batch/setup_job.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+
+import polars as pl
+from azure.batch import models
+from azuretools.auth import EnvCredentialHandler
+from azuretools.client import get_batch_service_client
+from azuretools.job import create_job_if_not_exists
+from azuretools.task import get_container_settings, get_task_config
+
+
+def main(job_id, pool_id, container_image) -> None:
+    creds = EnvCredentialHandler()
+    client = get_batch_service_client(creds)
+    job = models.JobAddParameter(
+        id=job_id,
+        pool_info=models.PoolInformation(pool_id=pool_id),
+    )
+    create_job_if_not_exists(client, job, verbose=True)
+
+    container_settings = get_container_settings(
+        container_image,
+        working_directory="containerImageDefault",
+        mount_pairs=[
+            {
+                "source": "nssp-etl",
+                "target": "/pyrenew-hew/nssp_demo/nssp-etl",
+            },
+            {
+                "source": "prod-param-estimates",
+                "target": "/pyrenew-hew/nssp_demo/params",
+            },
+            {
+                "source": "pyrenew-test-output",
+                "target": "/pyrenew-hew/nssp_demo/private_data",
+            },
+        ],
+    )
+
+    base_call = (
+        "/bin/bash -c '"
+        "python nssp_demo/forecast_state.py "
+        "--disease {disease} "
+        "--state {state} "
+        "--n-training-days 365 "
+        "--n-warmup 1000 "
+        "--n-samples 500 "
+        "--nssp-data-dir nssp_demo/nssp-etl/gold "
+        "--param-data-dir nssp_demo/params "
+        "--output-data-dir nssp_demo/private_data"
+        "'"
+    )
+
+    states = pl.read_csv(
+        "https://raw.githubusercontent.com/k5cents/usa/"
+        "refs/heads/master/data-raw/states.csv"
+    )
```

Comment:
I would replace this for a hashed version (permanent link) to that content. Ideally, I would also point to a file within pyrenew-hew.

Reply:
Long term I think we should point at https://www2.census.gov/geo/docs/reference/state.txt or a vintage thereof.

Reply:
That does not included population size, which is what we need.

Reply:
Oops, this one https://www2.census.gov/geo/docs/reference/cenpop2020/CenPop2020_Mean_ST.txt